esdt-nft-marketplace/src/token_whitelist.rs

```diff
@@ -0,0 +1,39 @@
+elrond_wasm::imports!();
+
+#[elrond_wasm::module]
+pub trait TokenWhitelistModule {
+    #[only_owner]
+    #[endpoint(addTokensToWhitelist)]
+    fn add_tokens_to_whitelist(&self, tokens_to_add: MultiValueEncoded<EgldOrEsdtTokenIdentifier>) {
+        let mut whitelisted_tokens_mapper = self.whitelisted_tokens();
+        for token_id in tokens_to_add {
+            whitelisted_tokens_mapper.insert(token_id);
+        }
+    }
+
+    #[only_owner]
+    #[endpoint(removeTokensFromWhitelist)]
+    fn remove_tokens_from_whitelist(
+        &self,
+        tokens_to_remove: MultiValueEncoded<EgldOrEsdtTokenIdentifier>,
+    ) {
+        let mut whitelisted_tokens_mapper = self.whitelisted_tokens();
+        for token_id in tokens_to_remove {
+            whitelisted_tokens_mapper.swap_remove(&token_id);
```

Comment:
Explicitly ignore the result.
```suggestion
            let _ = whitelisted_tokens_mapper.swap_remove(&token_id);
```